client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -176,10 +185,12 @@ const CheckoutThankYou = React.createClass( {
 
 	render() {
 		let purchases = null,
+			failedPurchases = null,
 			wasJetpackPlanPurchased = false,
 			wasOnlyDotcomPlanPurchased = false;
 		if ( this.isDataLoaded() && ! this.isGenericReceipt() ) {
 			purchases = getPurchases( this.props );
+			failedPurchases = getFailedPurchases(this.props );
```

Comment:
Oh, the missing space! Oh, how could you do this to me!? 😩 The whole code is ruined! _Ruined_!